src/main/java/util/LottoNumberConvertor.java

```diff
@@ -0,0 +1,20 @@
+package util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoNumberConvertor {
+
+    private static final String SPLIT_CHARACTERS = ", ";
+
+    public static List<Integer> convertLottoNumbers(final String lottoNumbers) {
```

Comment:
```suggestion
    public static List<Integer> convert(final String lottoNumbers) {
```

이런 메서드명은 어떠신가요?
클래스명을 통해 로또 번호에 대한 컨버팅이 이루어진다는 맥락을 이해할 수 있어서, 메서드명은 행위만 드러나게 적어도 좋을 것 같아요.

Reply:
아하!! 저는 최대한 모든 메소드에 의미를 담고 싶었던 것인데 클래스명에서 이미 구별을 했으면 괜찮은 거 같군요! 감사합니다 : )